test/swarming/bot-scripts/benchmark.py

```diff
@@ -45,10 +45,12 @@ def main():
 
     #### Call benchmark command
     gapit = os.path.join(agi_dir, 'gapit')
+    dumptrace = os.path.join(out_dir, 'benchmark.txt')
```

Comment:
Check if the extension is required to be .out for any reason. Otherwise good to go.

Reply:
I've updated to 'benchmark.systrace'